### PR TITLE
Added the change tracking module.

### PR DESCRIPTION
**Pull request by lloydlimmsft**
I still could not get it working - need someone to test that what I did is correct?

### CODE REVIEW

**Comment by sebassem** [comment]
> I still could not get it working - need someone to test that what I did is correct?

thank you @lloydlimmsft , we want to use azure policy instead of manually enabling every machine. You would just need to assign the _(ArcBox) Enable ChangeTracking and Inventory for Arc-enabled machines_ policy on the resource group and show how to get the _arcbox-ama-ct-dcr_ resource Id and provide it in the policy. Also if you can use the dark mode in Azure that would make all screenshot consistent :)

**Comment by lloydlimmsft** [comment]
Modified the screenshots to be dark instead of light now.
Please review - I've done what I could, even though it still does not work.

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 1]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
 
 #### Task 2
 
-### Module 5: Monitor changes to your Azure Arc-enabled servers using Change tracking and inventory
+### Module 5: Monitor changes to your Azure Arc-enabled servers using Change Tracking and Inventory
 
 #### Module overview
 
+Change Tracking and Inventory is an inbuilt Azure service, provided by Azure Automation. The old version uses the Log Analytics agent, while the new (preview) version uses the Azure Monitor Agent (AMA). The training here is taken from https://learn.microsoft.com/en-us/azure/automation/change-tracking/overview-monitoring-agent?tabs=win-az-vm
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:892]:
```suggestion
Change Tracking and Inventory is an built-in Azure service, provided by Azure Automation. The old version uses the Log Analytics agent, while the new (preview) version uses the Azure Monitor Agent (AMA).

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 2]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+### Module 5: Monitor changes to your Azure Arc-enabled servers using Change Tracking and Inventory
 
 #### Module overview
 
+Change Tracking and Inventory is an inbuilt Azure service, provided by Azure Automation. The old version uses the Log Analytics agent, while the new (preview) version uses the Azure Monitor Agent (AMA). The training here is taken from https://learn.microsoft.com/en-us/azure/automation/change-tracking/overview-monitoring-agent?tabs=win-az-vm
+
+#### Prerequisites
+
+The following are required for this module to function:
+1. Ensure that the servers are already provisioned in Arc.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:897]:
```suggestion
1. Ensure that the servers are already on-boarded to Azure Arc.

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 3]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+2. Ensure that the AMA is already provisioned on every server.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:898]:
```suggestion
2. Ensure that the Azure Monitor agent (AMA) is already deployed on every Arc-enabled server.

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 4]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+
+#### Current Limitations
+
+The following table lists the current limitations for Change Tracking And Inventory
+https://learn.microsoft.com/en-us/azure/automation/change-tracking/overview-monitoring-agent?tabs=win-az-vm#current-limitations
+
+Ensure that you have the correct region mappings for Azure Automation account and Log Analytics workspace as not all regions support both:
+https://learn.microsoft.com/en-us/azure/automation/how-to/region-mappings
+
 #### Task 1
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:913]:
```suggestion
#### Task 1: Creating an Automation account

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 5]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+Creating an Automation Account
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:914]:
```suggestion
```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 6]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+
+First create an automation account in Azure in the same location as Arcbox - ensure that the region mapping is supported as in the table above.
+```powershell
+$automationaccountname = "<your automation account name>"
+$location = "<your azure location>"
+$resourcegroupname = "<Your resource group name>"
+az automation account create --automation-account-name $automationaccountname --location $location --sku "Free" --resource-group $resourcegroupname
+```
+
+The Automation Account must also enable Change Tracking and Inventory by linking to the ArcBox Log Analytics workspace crated previously.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:924]: No need for this point as we are using Azure Policy to enable change tracking

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 7]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+![Screenshot showing how to enable inventory in Automation Account](./changetracking-ct-enabled.png)
 
 #### Task 2
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:927]:
```suggestion
#### Task 2: Enabling Change Tracking and Inventory

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 8]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+Enabling Change Tracking and Inventory
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:928]:
```suggestion
```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 9]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+
+```note
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:930]:
```suggestion
>NOTE: 
```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 10]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+For Azure Arc - you must then deploy a special policy and create a Data Collection Rule (DCR) to specify the data collection characteristics.
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:931]:
```suggestion
>For Azure Arc - you must then deploy a special policy and create a Data Collection Rule (DCR) to specify the data collection characteristics.

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 11]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+Follow the link here:
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:932]:
```suggestion
>Follow the link here:

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 12]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+https://learn.microsoft.com/en-us/azure/automation/change-tracking/enable-vms-monitoring-agent?tabs=multiplevms%2Carcvm
+
+The DCR will have already been deployed as part of the setup for this levelup,
+but you will need to know where to do this for your own environments in future.
+The policy deployment is done below.
+```
+
+You will need to deploy an Azure Initiative to enable Change Tracking on ARC enabled Virtual Machines.
+
+- In the Azure portal, search for the text "Preview" and for a definition type of "Initiative".
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:942]: Try to use the same steps as Azure Monitor, to first search for policy then the policy itself. Don;t use this policy, instead use the one the starts with (ArcBox), this is the new policy

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 13]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+
+    ![Screenshot showing assigning the policy to the right scope](./changetracking-assignpolicy2.png)
+
+- After validating the scope, click "Next" twice to navigate to the parameters tab.
+
+- To get the "Data Collection Rule" resource Id,  run the following CLI command
+
+
+```shell
+az resource show --name "MSVMI-ama-vmi-default-dcr" `
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:964]: This is the vminsights dcr not the change tracking one

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 14]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+                 --query id `
+                 --output tsv
+```
+
+- You can also find the "Data Collection Rule" resource Id from the Azure portal. Search for the _MSVMI-ama-vmi-default-dcr_ data collection rule.
+
+- Then click create for the initiave to be assigned to the _arcbox_ resource group.
+- Once it has been assigned, copy the assignmentID, as you will need it in the next part.
+
+![Screenshot showing the policyassignment](./changetracking-assignmentid.png)
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:976]: the screenshot has a red line, not sure if you meant to hide the subscription id

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 15]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+
+#to see the status of the remediation task.
+```
+You can see the processing state of each rememdiation task by clicking on the _Remediation_ tab
+
+![Screenshot showing the policyassignment](./changetracking-forcingremediation.png)
+
+Please be patient as it takes a while for onboarding to work.
+
+#### Task 3
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:1016]:
```suggestion
#### Task 3: Using Change Tracking

```

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md [hunk 16]
```diff
@@ -885,13 +885,188 @@ if ($roleDefinitionIds.Count -gt 0)
...
+
+Using Change Tracking
```
Comment (sebassem) [inline docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:1018]:
```suggestion
```

**Review by sebassem — CHANGES_REQUESTED** [review]

File: docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md (hunk 14 above)
Comment (lloydlimmsft) [reply docs/azure_arc_jumpstart/azure_arc_servers/levelup/_index.md:1008]: That is for the az command on the next part - to programmaically create remediation tasks for the policies.

**Pull request merged**